docs/multicluster_test_example.md

```diff
@@ -0,0 +1,112 @@
+# Multi-cluster test example
+
+The following example and notes are based on the main
+[PR#4875](https://github.com/red-hat-storage/ocs-ci/pull/4875) for this feature.
+
+## Example test
+
+Following example is simple test which do basically nothing, just shows how to
+switch between cluster configuration contexts and how to access configuration
+of particular cluster without switching context to it.
+
+```python
+import logging
+import os
+
+from ocs_ci.utility.utils import exec_cmd
+
+from ocs_ci.framework import config
+
+log = logging.getLogger(__name__)
+
+
+def test_multicluster_dahorak():
+    # print number of clusters to the log
+    log.info(f"Number of clusters: {config.nclusters}")
+
+    # iterate over the clusters and perform some actions on each cluster
+    for i in range(config.nclusters):
+        # switch context to the i-th cluster
+        config.switch_ctx(i)
+        # print the cluster name of the selected cluster
+        log.info(f"CLUSTER_NAME: {config.ENV_DATA['cluster_name']}")
+        # run oc command against the selected cluster
+        log.info(exec_cmd("oc version").stdout)
+        # print kubeconfig path of the selected cluster
+        log.info(os.environ["KUBECONFIG"])
+
+    # access configuration of particular cluster without switching context to it
+    log.info(f"Cluster name of first cluster: {config.clusters[0].ENV_DATA['cluster_name']}")
+    log.info(f"Cluster name of second cluster: {config.clusters[1].ENV_DATA['cluster_name']}")
+```
+
+## Execution of the example test
+
+See also [Running tests on multicluster environment section in Usage
+doc](usage.md#running-tests-on-multicluster-environment).
+
+I saved the above example test into `tests/test_multicluster_dahorak.py` file.
+Additionally I have two clusters and related cluster dirs:
+
+* cluster: `dahorak-test1`, cluster path: `../clusters/dahorak-test1/`
+* cluster: `dahorak-test2`, cluster path: `../clusters/dahorak-test2/`
+
+(Each cluster dir contains only `auth/kubeconfig` directory/file.)
+
+Now I'll run the test via following command:
+
+```bash
+$ run-ci multicluster 2 tests/test_multicluster_dahorak.py \
+    --ocsci-conf ocsci_conf.yaml --ocsci-conf conf/ocsci/skip_ocs_deploy.yaml \
```

Comment:
Do we really want to confuse people here with skip_ocs_deploy.yaml file?  I would only mention that in the note below that you run this example with this to avoid console output. BTW you might also use --dev-mode to avoid some of the collection info and others. 

Here people can think that it should be used like this if they will miss the note below that it was for such purpose.

Reply:
Good point, removed it in [b39429a](https://github.com/red-hat-storage/ocs-ci/pull/5415/commits/b39429a0b8c7820eb9f42c2dabae16ecce9c104e) and update the description.

I think I tried `--dev-mode`, but it didn't work as good as this :).